src/mesh/meshblock.cpp

```diff
@@ -143,38 +144,37 @@ void MeshBlock::Initialize(int igid, int ilid, LogicalLocation iloc,
   precon = std::make_unique<Reconstruction>(shared_from_this(), pin);
 
   // Add field properties data
+  // TOOD(JMM): Should packages be resolved for state descriptors in
+  // properties?
```

Comment:
Actually, I think we should just remove `properties`.  This is obviously not for this PR, but I doubt anyone will object.